scripts/toaster_configuration

```diff
@@ -218,13 +218,17 @@ if [ "\$#" -ne 1 ]; then
 else
   if [ "\$1" == "start" ]; then
     export CUSTOM_XML_ONLY=1
+    export OE_ROOT="$POKYDIR"
```

Comment:
We are not using poky distro now. Is it wise to rename this variable?

Reply:
@Noor-Ahsan, yes that would be wise but then the whole script requires some attention :) because there are various parts that can be overhauled.

Reply:
Yeah but we removed all poky related varaibles from meta-mentor. I was thinking atleast we update poky related var in this script too.

Reply:
@Noor-Ahsan, agreed. I'll push the update shortly.

Reply:
thanks